# 3D_Metalloproteome_Ecoli ( Gene-Protein Structure-Metal/Mutations (GPS-2M))
Integration of 3D Metalloproteins with ALE Mutations

![Graphical Abstract](https://github.com/user-attachments/assets/2d930575-c409-4ad7-8783-88781992b7e5)

Data-centric approaches that describe the functional and three-dimensional characteristics of the prokaryotic metalloproteome are limited, as are mechanistic approaches exploring the relationship between metal-binding properties and mutations. Analytics upon the reconstructed E. coli Gene-Protein Structure-Metal/Mutations (GPS-2M) dataset addresses these gaps by identifying spatial relationships between mutations on metalloproteins and their potential regulatory or metabolic impacts. GPS-2M integrates protein structures from protein crystallography data of known metal-binding positions and meta-analysis of adaptive laboratory evolution (ALE) and long-term evolutionary experiments (LTEE). This combination reveals trends in amino acid property changes with metal binding sites resulting from mutations. Investigation of GPS-2M identified spatial relationships between metal ions, amino acid residues, and ALE mutations within protein structures elucidates the prevalence of close-range interactions crucial for direct interactions, thus informing potential targets for protein design. Associating mutations with metal-binding sites has led to the identification of specific functional pathways, transcription factors, post-translational modifications, and downstream mutation targets associated with metalloproteins, which are of interest in microbial strain design. Specifically, we identified a range of close mutations to multiple metal-binding sites on proteins within a 3-5 Ã… range, suggesting important impacts of laboratory evolution upon the coordination between metal ions and their residues. This investigation, guided by specific ALE experiments, unveiled prime targets for strain optimization within E. coli. Mutations clustered around zinc-binding proteins PyrC, the mercury-binding uvrD, and molybdenum-binding narG, among numerous others, pinpointing optimal target sites for enhancing strain design efficacy. The proximity of these mutations underscores their potential significance in maintaining physiological homeostasis. Our data-driven approach deepens the understanding of the structural and evolutionary dynamics of the E. coli metalloproteome, with implications for scrutinizing mutated metal-binding sites for developing evolutionary-inspired biotechnology.
